### ISSUE TITLE
Is this working with calendarize_pages?

### ISSUE THREAD

**Issue by oliverschloebe** [cdaecke/md_fullcalendar#22]
Howdy,

I do have `calendarize`, `calendarize_pages` and `md_fullcalendar` installed but I'm not seeing any entries on the calendar even though I added calendarize data to a page.

Anyone got this working?
Thanks!

**Comment by cdaecke** [comment]
No, entries of `ext:calendarize_pages` are not supported in `ext:md_fullcalendar` up to now.

Pull requests are very welcome!
